build.proj

```diff
@@ -1,30 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  
+
```

Comment:
@markcowl yes because in last couple of commits, the tabs are not set right in our file, so formatted the entire file.